staging/src/k8s.io/client-go/transport/websocket/client.go

```diff
@@ -61,42 +62,12 @@ type ConnectionHolder interface {
 	Connection() *gwebsocket.Conn
 }
 
-// RoundTripper knows how to establish a connection to a remote WebSocket endpoint and make it available for use.
-// RoundTripper must not be reused.
-type RoundTripper struct {
-	// TLSConfig holds the TLS configuration settings to use when connecting
-	// to the remote server.
-	TLSConfig *tls.Config
-
-	// Proxier specifies a function to return a proxy for a given
-	// Request. If the function returns a non-nil error, the
-	// request is aborted with the provided error.
-	// If Proxy is nil or returns a nil *URL, no proxy is used.
-	Proxier func(req *http.Request) (*url.URL, error)
-
-	// Conn holds the WebSocket connection after a round trip.
-	Conn *gwebsocket.Conn
-}
-
-// Connection returns the stored websocket connection.
-func (rt *RoundTripper) Connection() *gwebsocket.Conn {
-	return rt.Conn
-}
-
-// DataBufferSize returns the size of buffers for the
-// websocket connection.
-func (rt *RoundTripper) DataBufferSize() int {
-	return 32 * 1024
-}
-
-// TLSClientConfig implements pkg/util/net.TLSClientConfigHolder.
-func (rt *RoundTripper) TLSClientConfig() *tls.Config {
-	return rt.TLSConfig
+type wsRoundtripper struct {
```

Comment:
don't export the custom roundtripper as it is not really a roundtripper https://github.com/kubernetes/kubernetes/pull/130364/files#r1966585753